### PR TITLE
changing services to match template

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

File: _posts/docker/2015-05-25-services.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ Some great examples of how and why we based `codeship-services.yml` on Docker Co
 
 **Note**: There are a few Docker Compose features not yet supported by `codeship-services.yml`. We will eventually be at full Docker Compose parity, but until then you can use most features of Docker Compose in your `codeship-services.yml` file with little to no modification. For a list of non-supporter Docker Compose features, see below in this article.
 
+## Prerequisites
```
Comment (mlocher) [inline _posts/docker/2015-05-25-services.md]: That's not really correct. There are no prerequisites (other than maybe an editor) unless you want to use `jet` locally. If you're only going to use the hosted platform there's nothing to do before writing the services file.

Comment (ethangj) [inline _posts/docker/2015-05-25-services.md]: @mlocher It says to have either installed Jet locally or have configure a project on Codeship. You will have to have done one or the other for the Services file to work.

Comment (mlocher) [inline _posts/docker/2015-05-25-services.md]: Yeah, I somehow missed the second part of the sentence :(

**Comment by mlocher** [comment]
:shipit:

File: _posts/docker/2015-05-25-services.md (hunk 1 above)
Comment (ethangj) [inline _posts/docker/2015-05-25-services.md]: @mlocher Hmm, do you have any ideas on how we could rewrite it to make it more visible or clear? If you missed it other people might, too.

Comment (mlocher) [inline _posts/docker/2015-05-25-services.md]: I'm just ready for the weekend, the text is clear enough. My fault :(

**Pull request merged**